Assets/Shopify/UIToolkit/ShopControllers/Classes/CartController.cs

```diff
@@ -25,6 +25,7 @@ public class PurchaseFailedEvent : UnityEvent<ShopifyError> {}
 
         public void SetCart(Cart cart) {
             Cart = cart;
+            Cart.LineItems.OnChange += OnLineItemsChange;
```

Comment:
Would it be worth being a bit more defensive here, and making sure that the event is only added once? Might help to prevent any bugs cropping up in the future.

Reply:
I'm gonna make it so that `Cart` can only be set through the constructor.